Managed Services/managed-os-operating-system-patching-and-update-processes.md

```diff
@@ -3,24 +3,59 @@
+
+As part of our managed cloud service, CenturyLink will work with your designated contact to install patches in agreed-upon maintenance windows.  Please contact the CenturyLink Client Service Center at 1-888-638-6771 and choose menu option 2 to submit your request for a maintenance activity.
+
+Since CenturyLink is often not responsible for the customer’s entire application tier, CenturyLink cannot be responsible for any adverse effects from patches that are installed at the request of the customer.
+
+#### CenturyLink Requesitioned Patching
+
+This includes keeping the system current with all patches to help prevent security compromises or operational reliability issues. CenturyLink will, from time to time, schedule the installation of system patches as deemed appropriate by our Solution Engineers. We will schedule the installation with you in advance. This will allow both parties to prepare for the patching, as well as provide ample time for discussion.
+
+
+### Advanced Patching
+
+Customized Patch Policies (environment & server specific) are not included with CenturyLink Cloud Managed OS. Please contact your CenturyLink sales representative to set up a special engagement.
+
+### OS Updates
+
+On a monthly basis, CenturyLink Cloud product engineers review all vendor-recommended patches made available by operating system software vendors. A subset of these patches and updates are identified by the OS vendor as **Critical** or **Important** to installed operating systems. The most recent version of our OS is given these updates and is then re-certified by CenturyLink before it is made available to be implemented with new server builds.
+
+### Managed Server Updates
+
+As a part of making a server managed, more updates may automatically be applied to the server. Please review the articles regarding [RedHat](..Managed Services/what-happens-when-RedHat-servers-are-made-managed.md) and [Windows](..Managed Services/what-happens-when-windows-servers-are-made-managed.md) to gain a general understanding of when these updates may be applied.
+
+### Service Packs and Sub Releases
+
+All service packs are evaluated and tested thoroughly on the standard CenturyLink Cloud Server configurations. Integration of the service pack into the CenturyLink standard operating system build only occurs once testing has proven its stability and performance benefits. All applicable servers will be automatically configured with this new base build after it has completed certification.
+
+CenturyLink intends to keep in production current level N-2 to the extent that the underlying vendor is still providing patching and support. When underlying vendor support ceases to exist, CenturyLink  will be liable for best effort service and patching up to the last available support structure. Existing production servers, if requested by the customer, will be upgraded during maintenance windows that will be coordinated with the customer.
+
+### Anti-Virus Protection
```

Comment:
is there a reason this whole statement on AV is in a managed OS patching KB.  i don't see how they are related in any way.  they are completely independent in my view.  but let me know.